Shopify/shopify-cli
# Configureable extension renderer version

### WHY are these changes introduced?

To enable configuring the extension renderer version on a per extension basis.

### WHAT is this pull request doing?

Passing down the renderer version as part of the extension config from the Ruby CLI to the go binary.

### How to test your changes?

To test these changes, you require [Shopify/shopify-cli-extensions PR #233](https://github.com/Shopify/shopify-cli-extensions/pull/233). Once you've checkout the PR please run `make build`. You're now ready to switch into `shopify-cli` and excute the following commands:

```
rake extensions:symlink

mkdir -p tmp
cd tmp/

../bin/shopify config feature --enable extension_server_beta
../bin/shopify extension create --type checkout_ui_extension --name test-extension --template typescript-react
jq '.dependencies["@shopify/checkout-ui-extensions-react"]' test-extension/package.json
```

The expected output is `~> 0.14.0` and not `latest` as before.

### Post-release steps

None. This behaviour continues to be behind a feature flag.

### Update checklist
